python_ta/debug/accumulation_table.py

```diff
@@ -156,23 +195,27 @@ def _tabulate_data(self) -> None:
                 return
 
         try:
-            if self.output_format == "table":
-                table = tabulate.tabulate(
-                    iteration_dict,
-                    headers="keys",
-                    colalign=(*["left"] * len(iteration_dict),),
-                    disable_numparse=True,
-                    missingval="None",
-                )
-                file_io.write(table)
-                file_io.write("\n")
-            else:
-                csv_preformat = [
-                    dict(zip(iteration_dict.keys(), row)) for row in zip(*iteration_dict.values())
-                ]
-                writer = csv.DictWriter(file_io, fieldnames=iteration_dict.keys())
-                writer.writeheader()
-                writer.writerows(csv_preformat)
+            for lst_index in range(len(self.loops)):
+                iteration_dict = self._create_iteration_dict(lst_index)
+                if self.output_format == "table":
+                    table = tabulate.tabulate(
+                        iteration_dict,
+                        headers="keys",
+                        colalign=(*["left"] * len(iteration_dict),),
+                        disable_numparse=True,
+                        missingval="None",
+                    )
+                    file_io.write("\n")
```

Comment:
I'm fine adding an extra newline, but it should only be between loop tables, and _not_ precede the first table.

By the way, this is also reflected in the documentation: after addressing this comment, you should also remove the extra blank line from the sample output for the new examples you added.